apps/web/src/design-system/sidebar/Sidebar.tsx

```diff
@@ -123,6 +122,7 @@ export const Sidebar = ({
       withinPortal={false}
       trapFocus={false}
       data-expanded={isExpanded}
+      zIndex={900}
```

Comment:
Can we just pass the `zIndex` as the prop to the Sidebar or allow to override `styles`?
I'm worried that we might break it in some other places.